### PR TITLE
Enable 120 character limit per line

### PR DESCRIPTION
**Pull request by vinodlala**

### CODE REVIEW

File: spicerack-styleguide/rubocop.yml [hunk 1]
```diff
@@ -415,7 +415,7 @@ Metrics/CyclomaticComplexity:
   Enabled: true
   Max: 6
 Metrics/LineLength:
-  Enabled: false
+  Enabled: true
```
Comment (jkminneti) [inline spicerack-styleguide/rubocop.yml]: 😭

Comment (tbconroy) [reply spicerack-styleguide/rubocop.yml]: Apparently this guy has unlimited screen space ☝️ :trollface:

Comment (tbconroy) [reply spicerack-styleguide/rubocop.yml]: Apparently this guy has unlimited screen space ☝️ :trollface:

**Comment by MichelleMik** [comment]
codeclimate lol

**Comment by vinodlala** [comment]
> codeclimate lol

😭

**Comment by MichelleMik** [comment]
> > codeclimate lol
> 
> 😭

Not crazy about this rule. There are some cases where we might want to go a little bit over. @jkminneti brought up spec contexts having longer names etc. I also have that guidance line in my text editor and I usually try to respect it unless its like a few characters over.

**Comment by vinodlala** [comment]
> > > codeclimate lol
> > 
> > 
> > 😭
> 
> Not crazy about this rule. There are some cases where we might want to go a little bit over. @jkminneti brought up spec contexts having longer names etc. I also have that guidance line in my text editor and I usually try to respect it unless its like a few characters over.

Longer spec contexts names?  Use less words.

I would love to see a legit super long context name like that just so I can reword it.

Anything else where 120 characters are "needed"?

**Pull request closed without merging**